src/events.js

```diff
@@ -6,7 +6,7 @@ const START_HOUR = parseInt(process.env.HAMMERTIME_START_HOUR || '6', 10);
 const STOP_HOUR = parseInt(process.env.HAMMERTIME_STOP_HOUR || '19', 10);
 
 function getCronHour(hour, zone) {
-  return DateTime.fromObject({ hour }, zone).setZone('UTC').hour;
+  return DateTime.fromObject({ hour }, { zone: zone }).setZone('UTC').hour;
```

Comment:
nit: `{ zone: zone }` does the same as `{ zone }`

Particularly noticeable right next the `{ hour }`